src/ui/toolbox/menu.js

```diff
@@ -55,6 +55,7 @@ var menuComponentApi = {
      */
     initMenu: function initMenu() {
         this.id = this.config.control;
+        this.typeNav = this.config.navType ? this.config.navType : 'fromLast';
```

Comment:
i think `navType` is more suitable variable name. Also could you please add some documentation for this configuration option

Reply:
fixed